app/filtering.js

```diff
@@ -497,7 +497,7 @@ function registerPermissionHandler (session, partition) {
 
       // Display 'Brave Browser' if the origin is null; ex: when a mailto: link
       // is opened in a new tab via right-click
-      const message = locale.translation('permissionMessage').replace(/{{\s*host\s*}}/, origin || 'Brave Browser').replace(/{{\s*permission\s*}}/, permissions[permission].action)
+      const message = locale.translation('permissionMessage').replace(/{{\s*host\s*}}/, getOrigin(requestingUrl) || 'Brave Browser').replace(/{{\s*permission\s*}}/, permissions[permission].action)
```

Comment:
Do you have an example of where the `setPermissionRequestHandler` handler gets called from? I see the definition in Muon, but am not sure when the handler is called?

Reply:
Looking at the code in Muon...
`mainFrameOrigin` is the `current_origin`
`requestingUrl` maps to `requesting_origin`

Super glad you fixed this mismatch issue 😄